spot-client/src/spot-remote/ui/views/remote-control.js

```diff
@@ -68,7 +70,7 @@ export class RemoteControl extends React.PureComponent {
      * @returns {ReactElement}
      */
     _getView() {
-        if (!this.props.isConnectedToSpot) {
+        if (this.props.isConnectionEstablished && !this.props.isConnectedToSpot) {
```

Comment:
Could isConnectionEstablished and isConnectedToSpot be combined into one prop instead? Maybe just a "loading" or "isConnecting" prop.